portafly/src/i18n/I18nProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Polyglot from 'node-polyglot'
+
+const polyglot = new Polyglot()
+const I18nContext = React.createContext(polyglot)
+
+interface Props {
+  children: React.ReactNode
+}
+
+const I18nProvider = (props: Props)=> {
```

Comment:
No need to declare custom props:
```suggestion
const I18nProvider: React.FunctionComponent = (props)=> {
```

Reply:
In fact we should discuss if we're using `Props` like we do in porta OR `React.FunctionComponent` like @didierofrivia has been doing.

Reply:
I like to define  `Props` when there's more than one property used